docgen/parser/model.go

```diff
@@ -16,7 +17,6 @@ type FileDescWrapper struct {
 // File is a proto parsed file.
 type File struct {
 	Swagger  *options.Swagger
-	Messages map[string]*Message
```

Comment:
this is not needed anymore in the new doc (since the messages are in Services)

enums are still needed, because they are in doc page annex; I will remove them in a later diff